exporter_test.go

```diff
@@ -221,6 +270,34 @@ func getImageFile(image v1.Image, layerDigest, path string) (string, error) {
 	return getLayerFile(layer, path)
 }
 
+func getImageFileUidGid(image v1.Image, layerDigest, path string) (int, int, error) {
```

Comment:
How about:
```
getImageFileOwner(image v1.Image, layerDigest, path string) (uid int, gid int, err error)
```

Reply:
Test helper function name changed in new commit